gobblin-iceberg/build.gradle

```diff
@@ -56,6 +56,7 @@ dependencies {
     testCompile project(path: ':gobblin-modules:gobblin-kafka-common', configuration: 'tests')
     testCompile externalDependency.testng
     testCompile externalDependency.mockito
+    testCompile externalDependency.powermock
```

Comment:
Mocking static method. Need to use this to mock the GobblinConstructorUtils and the FileSystem static calls

Reply:
Can you add these comments to code as well? so it's easily for us to track and understand in the future